app/scripts/contenteditor/migration/1_migration-task.js

```diff
@@ -3,29 +3,58 @@
 org.ekstep.contenteditor.migration = new (Class.extend({
 	migrationFlag: false,
 	contentBackup: undefined,
+	patch: [],
 	init: function () {
 		org.ekstep.contenteditor.api.addEventListener('content:migration:start', this.execute, this)
 	},
 	_startTime: undefined,
 	tasks: ['mediamigration_task', 'basestage_task', 'orderstage_task', 'scribblemigration_task', 'imagemigration_task', 'readalongmigration_task', 'assessmentmigration_task', 'eventsmigration_task', 'settagmigration_task'],
+	questionPatchs: ['questionsetfix1'],
```

Comment:
@pallakartheekreddy - Can we rename "questionsetfix1" as "patch_1" or may be 6 characters alphanumeric value?

Reply:
Yes, we can use `patch_1`.  list of patches I am maintaining an array to verify and run the migration patch.